build.gradle.kts

```diff
@@ -119,43 +117,22 @@ tasks {
     }
 }
 
-val sourcesJar by tasks.registering(Jar::class) {
-    dependsOn(JavaPlugin.CLASSES_TASK_NAME)
-    group = JavaBasePlugin.DOCUMENTATION_GROUP
-    description = "Assembles sources JAR"
-    archiveClassifier.set("sources")
-    from(sourceSets.main.get().allSource)
-}
-
 gradlePlugin {
+    website.set(webUrl)
+    vcsUrl.set(githubUrl)
     plugins {
         create("kotlinterPlugin") {
             id = pluginId
-            implementationClass = "org.jmailen.gradle.kotlinter.KotlinterPlugin"
-        }
-    }
-}
-
-pluginBundle {
-    website = webUrl
-    vcsUrl = githubUrl
-    description = project.description
-    tags = listOf("kotlin", "ktlint", "lint", "format", "style", "android")
-
-    plugins {
-        named("kotlinterPlugin") {
             displayName = "Kotlin Lint plugin"
+            description = project.description
+            tags.addAll(listOf("kotlin", "ktlint", "lint", "format", "style", "android"))
+            implementationClass = "org.jmailen.gradle.kotlinter.KotlinterPlugin"
```

Comment:
I followed the guide here https://docs.gradle.org/current/userguide/publishing_gradle_plugins.html#custom-plugin-repositories
![image](https://user-images.githubusercontent.com/36954793/211162148-499ccaf0-c988-4e43-8ecd-b90f54444dec.png)